cli/src/commands/log.rs

```diff
@@ -76,11 +77,19 @@ pub(crate) fn cmd_log(
 ) -> Result<(), CommandError> {
     let workspace_command = command.workspace_helper(ui)?;
 
+    // if no args are passed, use default revset
+    // if only paths are passed, filter by paths
+    // if only a revset is passed, use the revset
+    // if paths and revset are passed, use an intersection of both
```

Comment:
nit: this comment is obvious from the code, so let's remove it.

Reply:
oops actually wanted to do that since I added the comments for the specific statements 👍🏻